rbx_binary/src/deserializer.rs

```diff
@@ -344,6 +348,29 @@ impl<R: Read> BinaryDeserializer<R> {
         Ok(())
     }
 
+    fn decode_sstr_chunk(&mut self, mut chunk: &[u8]) -> Result<(), InnerError> {
+        let version = chunk.read_le_u32()?;
+
+        if version != 0 {
+            return Err(InnerError::UnknownChunkVersion {
+                chunk_name: "PRNT",
+                version,
+            });
+        }
+
+        let num_entries = chunk.read_le_u32()?;
+
+        for i in 0..num_entries {
+            chunk.read_exact(&mut [0; 16])?; // We don't do anything with the hash.
+
+            let data = chunk.read_binary_string()?;
+
+            self.shared_strings.insert(i, SharedString::new(data));
```

Comment:
We should be able to get rid of the blank lines between statements in this loop. It's all pretty straight-line code.